doc/README.md

```diff
@@ -73,7 +74,7 @@ Flame doesn't bundle any network feature, which may be needed to write online mu
 If you are building a multipler game, here are some recommendations of packages/services:
 
 - [Nakama](https://github.com/Allan-Nava/nakama-flutter): Nakama is an open-source server designed
- to power modern games and apps.
+to power modern games and apps.
```

Comment:
Here there should actually be whitespace to align with the first line if the list item